Core/Domibus-MSH/src/main/java/eu/domibus/core/message/UserMessageDefaultService.java

```diff
@@ -82,6 +83,9 @@ public class UserMessageDefaultService implements UserMessageService {
     public static final int BATCH_SIZE = 100;
     static final String PAYLOAD_NAME = "PayloadName";
     static final String MIME_TYPE = "MimeType";
+    static final String BUSINESS_CONTENT_ATTACHMENT = "businessContentAttachment";
+    static final String BUSINESS_CONTENT_ATTACHMENT_ENABLED = "domibus.payload.business.content.attachment.enabled";
```

Comment:
Could this be replaced with `import static eu.domibus.api.property.DomibusPropertyMetadataManagerSPI.DOMIBUS_PAYLOAD_BUSINESS_CONTENT_ATTACHMENT_ENABLED;`? In that way, it would be enough to define the property name in one place only.

Reply:
Good idea! I placed the constants in a single file.